tv/components/RoomDisplay.tsx

```diff
@@ -11,6 +11,14 @@ type RoomProps = {
   nextSession: Session | null;
 };
 
+const handleCurrentSessionQrCodeClick = (currentSession: Session) => {
```

Comment:
These functions would be best located inside the `RoomDisplay` functional component. Their input parameter could be removed. The props could be used instead.